jpype/_pykeywords.py

```diff
@@ -16,20 +16,64 @@
 #
 # *****************************************************************************
 
+import typing
+
 # This is a superset of the keywords in Python.
 # We use this so that jpype is a bit more version independent.
-# Removing keywords from this list impacts the exposed interfaces, and therefore is a breaking change. 
-_KEYWORDS = set((
-    'False', 'None', 'True', 'and', 'as', 'assert', 'async',
-    'await', 'break', 'class', 'continue', 'def', 'del', 'elif', 'else',
-    'except', 'exec', 'finally', 'for', 'from', 'global', 'if', 'import',
-    'in', 'is', 'lambda', 'nonlocal', 'not', 'or', 'pass', 'print',
-    'raise', 'return', 'try', 'while', 'with', 'yield'
-))
+# Adding and removing keywords from this list impacts the exposed interfaces,
+# and therefore is technically a breaking change.
+_KEYWORDS = {
```

Comment:
This list is unchanged, but I split it into single lines to make it easier to add individual comments (if required), and to easily add/remove items without a major diff.